components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import { action } from "@storybook/addon-actions";
+import * as React from "react";
+import styled from "styled-components";
+import Card from "./Card";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+
+export default {
+  title: "Card",
+  component: Card
+};
+
+export const example = () => (
+  <Card style={{ height: "200px", width: "400px" }}>200 * 400</Card>
+);
+
+export const clickable = () => (
+  <Card onClick={action("onClick")} style={{ height: "200px", width: "400px" }}>
+    200 * 400
```

Comment:
Generally speaking, width comes before height.

```suggestion
    400 x 200
```